plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2973,7 +2974,7 @@ public Domain getDomain(final Connect conn, final String vmName) throws LibvirtE
     }
 
     private String executeBashScript(final String script) {
-        final Script command = new Script("/bin/bash", _timeout, s_logger);
+        final Script command = new Script("/bin/bash", _timeout.getMillis(), s_logger);
```

Comment:
Please consider adding an overridden version of the `Script(String, int, Logger)` constructor that accepts `Duration` to encapsulate this type conversion.